free-3.md

```diff
@@ -10,41 +10,91 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Some people use white noise to help them concentrate or fall asleep.
+Some people also have preferences based on the frequency spectrum used
+to generate a noise.
+For example, some prefer pink noise to white noise.
+The DSL would let people customize random noise similar to white noise.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL could make it easy for someone to generate random noise and customize it
+to their liking.
+It is easy to find white, pink, brown, blue, violet, and grey noise but it is
+harder to make noise from a different frequency spectrum.
 
```

Comment:
I question if a language is necessary over an application with a frequency slider but they both could work?!